docs/source/cuda/overview.rst

```diff
@@ -2,6 +2,36 @@
 Overview
 ========
 
+.. cuda-deprecated::
+
+.. _cuda-deprecation-status:
+
+CUDA Target Deprecation and maintenance status
```

Comment:
I've got a stray capitalisation here:

```suggestion
CUDA Target deprecation and maintenance status
```

(thinking of "CUDA Target" as a name and therefore keeping the "T").